### DISCUSSION TITLE
Rendered Markdown or HTML export support

### DISCUSSION THREAD

**Discussion by hughess** [evidence-dev/evidence#2349] [General]
**Feature Description**
Original idea from @aaronsteers:
> might prefer a lighter weight and more portable export format over pdf. Is there any thought around html and/or "rendered markdown" exports? Some applications I'm thinking of are to post one or more rendered output pages to a GitHub PR, as an attachment in an automated Slack post, or similar automated pushes in the format of messages/posts.

**Comment by sandstrom** [comment]
This would be great!

Ideally with charts / graphics as SVG.
